apps/trading-e2e/src/integration/deposits.feature

```diff
@@ -58,19 +58,50 @@ Feature: Deposits to vega wallet
   # Then I can see the deposit is Successfull
   # And Balance is updated to reflect deposit amount
 
-  Scenario: Validation errors
-    # wallet is connected on before hook so this step may no longer be required
-    # Given I connect my Ethereum wallet
-    When I submit a deposit with empty fields
-    Then I can see validation errors present
-    And I enter an invalid public key
+  Scenario: Empty form Validation errors
+    When I enter the following deposit details in deposit form
+      | asset  |  |
+      | to     |  |
+      | amount |  |
+    And I submit the form
+    Then I can see empty form validation errors present
+
+  Scenario: Invalid deposit Public Key Validation error
```

Comment:
There's inconsistent casing on the scenario titles throughout the feature file. It should be capitals on first word of the title and lower case throughout except for Names etc.